crates/router/src/core/payments/helpers.rs

```diff
@@ -105,7 +105,8 @@ pub fn create_identity_from_certificate_and_key(
     let certificate_key = String::from_utf8(decoded_certificate_key)
         .change_context(errors::ApiClientError::CertificateDecodeFailed)?;
 
-    reqwest::Identity::from_pkcs8_pem(certificate.as_bytes(), certificate_key.as_bytes())
+    let key_chain = format!("{}{}", certificate_key, certificate);
```

Comment:
I think we can move `create_identity_from_certificate_and_key()` and `create_certificate()` to `crates/router/src/services/api/client.rs` file.